sdk/provisioning/Azure.Provisioning/api/Azure.Provisioning.netstandard2.0.cs

```diff
@@ -75,7 +75,7 @@ internal BicepValue() { }
         public virtual bool IsEmpty { get { throw null; } }
         void Azure.Provisioning.IBicepValue.Assign(Azure.Provisioning.IBicepValue source) { }
         void Azure.Provisioning.IBicepValue.SetReadOnly() { }
-        public Azure.Provisioning.Expressions.BicepExpression Compile() { throw null; }
+        public virtual Azure.Provisioning.Expressions.BicepExpression Compile() { throw null; }
```

Comment:
In the implementation, you could see I add this virtual to make sure we could be possible to throw the exception when the item gets compiled.
We could add a `private proected virtual BicepExpression CompileCore()` in order not to change public API.